.github/workflows/release-v2.yml

```diff
@@ -33,10 +33,16 @@ jobs:
     outputs:
       release-tag: ${{ steps.resolve_tag.outputs.tag }}
     steps:
+      - uses: actions/checkout@v3
+
+      - name: Set reference
+        id: vars
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+
       - name: Resolve tag
         id: resolve_tag
         run: |
-          TAG=${{ inputs.tag != null && inputs.tag || github.ref }}
+          TAG=${{ inputs.tag != null && inputs.tag || steps.vars.outputs.tag }}
```

Comment:
I may be wrong, but wouldn't this be same as simple:

    TAG=${{ inputs.tag || steps.vars.outputs.tag }}

assuming Github action follows usual "falsy" logic of scripting languages.

Reply:
Could be, but I don't want to have a yet another failure here.. This is working imo..